CHANGELOG.md

```diff
@@ -71,6 +72,12 @@ how a consumer would use the library (e.g. adding unit tests, updating documenta
 - Now, when adding a cloud alias to a detection list user, such as during `departing-employee add`, it will remove the existing cloud alias if one exists.
     - Before, it would error and the cloud alias would not get added.
 
+### Added
+
+- `code42 devices list` option:
+    - `--include-legal-hold-membership` to add legal hold columns to the result output, printing the legal hold matter name and ID for any active device actively on legal hold
```

Comment:
can we rephrase `printing ....` to something like `prints the legalhold matter name and ID for any active device on legal hold`!? 

atleast `active device actively` seems off while reading.

Reply:
yes, that is terrible phrasing. Resolved with latest commit